pkg/registry/apis/dashboard/legacy/sql_dashboards.go

```diff
@@ -327,13 +320,20 @@ func (a *dashboardSqlAccess) scanRow(rows *sql.Rows, history bool) (*dashboardRo
 				if err != nil {
 					return nil, err
 				}
-				repo.Path = originPath
+				meta.SetSourceProperties(utils.SourceProperties{
+					Path:      originPath, // relative path within source
+					Checksum:  origin_hash.String,
+					Timestamp: origin_ts.Int64,
+				})
+				meta.SetManagerProperties(utils.ManagerProperties{
+					Kind:     utils.ManagerKindRepo, // classic?
```

Comment:
NIT – I think having a different kind here would help, because the new repo one has different semantic.